deny.toml

```diff
@@ -0,0 +1,30 @@
+# See docs: https://embarkstudios.github.io/cargo-deny/checks/index.html
+
+[advisories]
+unmaintained = "warn"
+vulnerability = "warn"
+yanked = "deny"
+notice = "deny"
+
+[bans]
+multiple-versions = "warn"
+wildcards = "deny"
+skip = [
+    { name = "itertools", version = "0.9.0" }, # criterion brings in two versions
+]
+
+[sources]
+unknown-registry = "deny"
+unknown-git = "deny"
+required-git-spec = "rev"
+
+[licenses]
+copyleft = "deny"
+
+# Run `cargo deny list` to see which crates use which license
+# and add them to this array if we accept them
+
+allow = ["MIT", "Apache-2.0", "Unlicense", "BSD-3-Clause"]
```

Comment:
Does this align with the license on this repo?

Reply:
Actually I guess adding Apache 2 as a dual licence wouldn't restrict which would be allow, so that should be fine :+1: